### ISSUE TITLE
Web help doesn't load in IE11

### ISSUE THREAD

**Issue by tjohnsonBCM** [zowe/imperative#393]
Launching the web help in IE11 results in a mostly blank page:
![image](https://user-images.githubusercontent.com/50887074/79116579-df958900-7d56-11ea-8a4b-28fec464bba5.png)

Although Microsoft discourages use of IE11 in favor of modern alternatives, it is still commonly used by some enterprises, and supported better by accessibility software than newer browsers. If it's not too difficult, web help should be made usable in IE11.
